config/ie/postCoreHoudiniInstall

```diff
@@ -42,7 +42,7 @@
 import sys
 
 import VersionControl 
-VersionControl.setVersion('IEBuild', '6.15.0') 
+VersionControl.setVersion('IEBuild', '6') 
```

Comment:
Oh, I released IEBuild as 6.24.0 since it was changing the behaviour of the function instead of fixing an existing behaviour. Is that okay?